source/Symbol/SwiftASTContext.cpp

```diff
@@ -390,6 +390,9 @@ class SwiftCStyleEnumDescriptor : public SwiftEnumDescriptor {
     m_nopayload_elems_bitmask =
         enum_impl_strategy.getBitMaskForNoPayloadElements();
 
+    if (enum_decl->isObjC())
```

Comment:
Does this not apply to C enums on Linux?

Reply:
So?

Reply:
AFAIK, they're still called `@objc` enums on Linux, even though they're technically C enums.